react/features/analytics/middleware.ts

```diff
@@ -104,6 +106,16 @@ MiddlewareRegistry.register(store => next => action => {
             // the user will be redirected to another page and new instance of
             // Analytics will be created and initialized.
             resetAnalytics();
+
+            const { dispatch } = store;
+
+            batch(() => {
+                dispatch({
+                    type: SET_INITIALIZED,
+                    value: false
+                });
+                dispatch({ type: CLEAR_INITIAL_PERMANENT_PROPERTIES });
```

Comment:
I think it intuitively makes sense to clear them when SET_INITIALIZED is dispatched with false, so I would remove the explicit action.

Reply:
Good idea!